src/App.js

```diff
@@ -8,29 +8,35 @@ import { mentors } from "./mentors.json";
 import "./App.css";
 
 const visibleMentorIds = mentors.map((mentor, index) => index);
+
+// Convert names into usable `#`-ids.
+const mentorNames = mentors.map(({ name }) =>
```

Comment:
Could I propose using `mentorHashes` instead? Just to keep it separate from the idea of the mentors' names.